### PR TITLE
[Doc] add more MBart and other doc

### PR DESCRIPTION
**Pull request by patil-suraj**
This PR 
1. adds example for MBart
2. adds MBart in pre_trained models list and readme (Pegasus was missing from readme, so also added that).

@sshleifer , @sgugger

### CODE REVIEW

**Comment by patil-suraj** [comment]
@sgugger  do you think it would be a good idea to add more fine-tuning info for MBart, since it requires input processed in a different way than other models as it is multilingual model ?

File: docs/source/index.rst [hunk 1]
```diff
@@ -126,7 +126,7 @@ conversion utilities for the following models:
     Min, Patrick Lewis, Ledell Wu, Sergey Edunov, Danqi Chen, and Wen-tau Yih.
 23. `Pegasus <https://github.com/google-research/pegasus>`_ (from Google) released with the paper `PEGASUS: Pre-training with Extracted Gap-sentences for Abstractive Summarization
     <https://arxiv.org/abs/1912.08777>`_ by Jingqing Zhang, Yao Zhao, Mohammad Saleh and Peter J. Liu.
-24. `MBart <https://github.com/pytorch/fairseq/tree/master/examples/mbart>`_ (from Facebook) released with the paper  `Multilingual Denoising Pre-training for Neural Machine Translation <https://arxiv.org/abs/2001.08210>`_ by Yinhan Liu, Jiatao Gu, Naman Goyal, Xian Li, Sergey Edunov
+24. `MBart <https://github.com/pytorch/fairseq/tree/master/examples/mbart>`_ (from Facebook) released with the paper  `Multilingual Denoising Pre-training for Neural Machine Translation <https://arxiv.org/abs/2001.08210>`_ by Yinhan Liu, Jiatao Gu, Naman Goyal, Xian Li, Sergey Edunov,
```
Comment (sgugger) [inline docs/source/index.rst:129]: It looks like the two lists got out of date with DPR (it should be in 22 in that list). Would you mind fixing this in your PR? Can do in another one if it's a problem.

**Review by sgugger — COMMENTED** [review]
Thanks for your PR, it looks good to me. If there is any specific behavior for MBart needed in the preprocessing, it should be documented, yes.

File: docs/source/index.rst (hunk 1 above)
Comment (patil-suraj) [reply docs/source/index.rst:129]: no worries, will fix it in this PR.

**Review by sshleifer — APPROVED** [review]

**Comment by patil-suraj** [comment]
@sshleifer ,@sgugger  added DPR in readme.

File: docs/source/index.rst (hunk 1 above)
Comment (sgugger) [reply docs/source/index.rst:129]: Thanks!

**Comment by sgugger** [comment]
Great! Thanks for the PR.

**Pull request merged**